timemachine/fe/atom_mapping.py

```diff
@@ -115,7 +115,8 @@ def get_cores(
 
     Additional notes
     ----------------
-    1) The returned cores are sorted in increasing order based on the rmsd of the alignment.
+    1) The returned cores are jointly sorted in increasing order based on the number of core-dummy bonds broken, the
+       number of valence values changed and the rmsd of the alignment.
     2) The number of cores atoms may vary slightly, but the number of mapped edges are the same.
```

Comment:
I had forgotten about this aspect of the atom mapping code, do we know how many atoms it differs by and if we should perhaps consider this in the sort (presumably fewer atoms will be selected based on the RMSD)?

Reply:
Not sure this is true, I added in an assertion here that cores are all of the same size and haven't been able to trigger it. @proteneer can you comment on this?